contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--2.2.0--2.3.0.sql

```diff
@@ -3,6 +3,43 @@
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
 
+CREATE OR REPLACE FUNCTION sys.babelfishpg_common_get_babel_server_collation_oid() RETURNS OID
+LANGUAGE C
+AS 'babelfishpg_common', 'get_babel_server_collation_oid';
+
+
+-- Set the default collation of given sys."typname" to oid corresponding to babelfishpg_tsql.server_collation_name
+CREATE OR REPLACE PROCEDURE sys.babelfish_update_collation_to_default(type_name varchar) AS
+$$
+DECLARE
+    coll_oid oid;
+BEGIN
+    SELECT typcollation INTO coll_oid FROM pg_catalog.pg_type WHERE typname = type_name AND typnamespace = (SELECT oid FROM pg_namespace WHERE nspname = 'sys');
+    IF coll_oid = 100 THEN
```

Comment:
Yes, it is safe to assume that. check https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/blob/BABEL_2_X_DEV__PG_14_X/src/include/catalog/pg_collation.dat#L15
But I made it to fetch it from pg_collation catalog instead of hard coded value.